pyanaconda/modules/user/user_kickstart.py

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
Please, rename this file to `kickstart.py`.

Reply:
Renamed.